Seasonal/Mogloween/Tricktown.cs

```diff
@@ -0,0 +1,75 @@
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreStory.cs
+using Skua.Core.Interfaces;
+
+public class Tricktown
+{
+    public IScriptInterface Bot => IScriptInterface.Instance;
+    public CoreBots Core => CoreBots.Instance;
+    public CoreStory Story = new();
+
+    public void ScriptMain(IScriptInterface bot)
+    {
+        Core.SetOptions();
+
+        StoryLine();
+
+        Core.SetOptions(false);
+    }
+
+    public void StoryLine()
+    {
+        if (Core.isCompletedBefore(8435))
```

Comment:
Change to 
if (Core.isCompletedBefore(8935) && !Core.IsSeasonalMapAvailable("tricktown"))
return;

seasonal map available function checks if this mao available or not as it is seasonal